src/app/visit/treatment.cpp

```diff
@@ -0,0 +1,29 @@
+#include "treatment.hpp"
+
+#include <map>
+#include <stdexcept>
+
+namespace
+{
+const std::map<Treatment, std::string> DentalTreatments = {
+    {Treatment::TeethCleaning, "Teeth Cleaning"},   {Treatment::RootCanal, "Root Canal"},
```

Comment:
In my opition we should provide lowercase strings and later on provide layer that will take care of formatting when needed